.circleci/config.yml

```diff
@@ -174,16 +174,46 @@ workflows:
   version: 2
   workflow:
     jobs:
-      - lintCheck
+      - lintCheck: 
+          filters:
+            tags:
+              only: /^v.*/
+            branches:
+              only: /.*/
```

Comment:
Does that mean every branch?

Is it possible not to mention that?
```suggestion
```

Reply:
yes it should be every branch.
I was thinking the same that i dont need that. but the pipeline didnt start when pushing a tag so i thought i might need to add that